tox.ini

```diff
@@ -78,3 +83,10 @@ basepython = python3
 skip_install = true
 commands =
     /bin/sh -c 'tar -czvf build_files.tar.gz build/'
+
+[test-deps]
+deps =
+# If you need to test on a certain branch, add @<branch-name> after .git
+    git+https://github.com/aws/amazon-braket-schemas-python.git
+    git+https://github.com/aws/amazon-braket-default-simulator-python.git
```

Comment:
Is the dependency on default simulator needed?

Reply:
Nope, good catch